lib/platform/bitbucket/index.js

```diff
@@ -76,6 +77,12 @@ async function getRepos(token, endpoint) {
   }
 }
 
+function getAuthor() {
+  logger.info('Getting bitbucket gitAuthor');
```

Comment:
I think these logger.info messages when we return null would just lead to confusion if users look at logs. Probably best not to log anything at all.

Reply:
makes sense :+1: